<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="girl1">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/girl1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="boy1">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>180</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/boy1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="boy2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>180</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/boy2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="playdrum">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="mytime">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="girl1_2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>180</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/girl1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="girl2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>180</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/girl2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="boy1_2">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>180</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/boy1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="drum">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>341</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/drum.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="startgame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/taikostart.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="startbutton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endbutton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>END</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restart">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RESTART</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endgame">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>END</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>50</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>310</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="button1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/button1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="button2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/button2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="smallgirl">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/smallgirl.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="forevergirl">
    <property name="geometry">
     <rect>
      <x>4400</x>
      <y>0</y>
      <width>107</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/forever.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="blue">
    <property name="geometry">
     <rect>
      <x>4400</x>
      <y>150</y>
      <width>800</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/blue.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="space1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/space2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="space2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/space1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="k1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/k1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="k2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/k2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="s2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/s2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="s1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/s1.png</pixmap>
    </property>
   </widget>
   <zorder>endgame</zorder>
   <zorder>restart</zorder>
   <zorder>playdrum</zorder>
   <zorder>score</zorder>
   <zorder>mytime</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>boy1</zorder>
   <zorder>boy1_2</zorder>
   <zorder>boy2</zorder>
   <zorder>girl1</zorder>
   <zorder>girl1_2</zorder>
   <zorder>girl2</zorder>
   <zorder>smallgirl</zorder>
   <zorder>blue</zorder>
   <zorder>forevergirl</zorder>
   <zorder>button2</zorder>
   <zorder>button1</zorder>
   <zorder>drum</zorder>
   <zorder>space2</zorder>
   <zorder>k2</zorder>
   <zorder>s2</zorder>
   <zorder>k1</zorder>
   <zorder>s1</zorder>
   <zorder>space1</zorder>
   <zorder>startgame</zorder>
   <zorder>startbutton</zorder>
   <zorder>endbutton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
